skiko/build-with-local-skia.sh

```diff
@@ -43,7 +43,7 @@ case $SKIA_TARGET in
     ;;
```

Comment:
It will be good if we can include Angle as a jar via `skiko-awt-runtime-angle-$platform`

Reply:
I'll see if I can figure this out